sonar-db/src/main/java/org/sonar/db/version/v55/CreateInitialSchema.java

```diff
@@ -0,0 +1,918 @@
+    VarcharColumnDef nameCol = newVarcharColumnDefBuilder().setColumnName("name").setLimit(100).setIsNullable(false).build();
+    context.execute(
+      new CreateTableBuilder(getDialect(), "quality_gates")
+        .addPkColumn(newIntegerColumnDefBuilder().setColumnName("id").setIsNullable(false).build(), AUTO_INCREMENT)
+        .addColumn(nameCol)
+        .addColumn(newTimestampColumnDefBuilder().setColumnName("created_at").setIsNullable(true).build())
+        .addColumn(newTimestampColumnDefBuilder().setColumnName("updated_at").setIsNullable(true).build())
+        .build());
+    addIndex(context, "quality_gates", "uniq_quality_gates", true, nameCol);
+  }
+
+  private void createQualityGateConditions(Context context) throws SQLException {
+    context.execute(
+      new CreateTableBuilder(getDialect(), "quality_gate_conditions")
+        .addPkColumn(newIntegerColumnDefBuilder().setColumnName("id").setIsNullable(false).build(), AUTO_INCREMENT)
+        .addColumn(newIntegerColumnDefBuilder().setColumnName("qgate_id").setIsNullable(true).build())
+        .addColumn(newIntegerColumnDefBuilder().setColumnName("metric_id").setIsNullable(true).build())
+        .addColumn(newIntegerColumnDefBuilder().setColumnName("period").setIsNullable(true).build())
+        .addColumn(newVarcharColumnDefBuilder().setColumnName("operator").setLimit(3).setIsNullable(true).build())
+        .addColumn(newVarcharColumnDefBuilder().setColumnName("value_error").setLimit(64).setIsNullable(true).build())
+        .addColumn(newVarcharColumnDefBuilder().setColumnName("value_warning").setLimit(64).setIsNullable(true).build())
+        .addColumn(newTimestampColumnDefBuilder().setColumnName("created_at").setIsNullable(true).build())
+        .addColumn(newTimestampColumnDefBuilder().setColumnName("updated_at").setIsNullable(true).build())
+        .build());
+  }
+
+  private void createProperties(Context context) throws SQLException {
+    VarcharColumnDef propKey = newVarcharColumnDefBuilder().setColumnName("prop_key").setLimit(512).setIsNullable(true).build();
+    context.execute(
+      new CreateTableBuilder(getDialect(), "properties")
+        // do not define as primary key on purpose -> already set in org.sonar.db.version.v61.CreateTableProperties2
+        .addColumn(newIntegerColumnDefBuilder().setColumnName("id").setIsNullable(false).build())
+        .addColumn(propKey)
+        .addColumn(newIntegerColumnDefBuilder().setColumnName("resource_id").setIsNullable(true).build())
+        .addColumn(newClobColumnDefBuilder().setColumnName("text_value").setIsNullable(true).build())
+        .addColumn(newIntegerColumnDefBuilder().setColumnName("user_id").setIsNullable(true).build())
+        .build());
+    addIndex(context, "properties", "properties_key", false, propKey);
+  }
+
+  private void createProjectLinks(Context context) throws SQLException {
+    context.execute(
+      new CreateTableBuilder(getDialect(), "project_links")
```

Comment:
you could factor `context.execute(new CreateTableBuilder(getDialect(), "XXXXX")` into something like `createTable(newTableBuilder("XXXX").[... no call to build...])`, a bit as you did for index creation